app/views/provider/admin/messages/inbox/index.html.slim

```diff
@@ -1,24 +1,32 @@
 - content_for :page_header_title, 'Inbox'
 
-= render 'provider/admin/messages/bulk_operations', scope: :received_messages, messages: @messages
+- if @messages.blank?
+  div class="pf-c-empty-state"
+    div class="pf-c-empty-state__content"
+      i class="fas fa-rocket pf-c-empty-state__icon" aria-hidden="true"
```

Comment:
I'm suggesting :inbox_tray: 
https://fontawesome.com/v4/icon/inbox

Reply:
Not saying I don't like it, but Patternfly seems to be rather strict about icons in empty states: 
>To maintain consistency, specific icons are mapped to specific empty state scenarios.

https://www.patternfly.org/components/empty-state/design-guidelines#icons

I will ask in their Slack

Reply:
yeah, but according to that page, :rocket: is for getting-started, which is not the case

also it says "You can replace this icon if you already have an application-specific graphic that you want to use instead." :grimacing: 

but if we stick to that info, looks like :mag: is the one to use.
But I'd still prefer the icons I suggested :sweat_smile:

Reply:
I confirmed with them these are just suggestions so we go with the inbox icon.